{{cookiecutter.project_name}}/Makefile

```
@@ -83,15 +90,18 @@ clean:
 	rm -rf __pycache__
 	find . -name "*.pyc" -exec rm -f {} \;
 
-test: venv_check install_dev
-	python manage.py test
+test: venv_check install_dev start-db
+	ACG_CONFIG=cdappconfig.json ${PYTHON_CMD} manage.py test
+	make stop-db
 
-smoke-test: venv_check install_dev
+smoke-test: venv_check install_dev start-db
 	@echo "Running smoke tests"
+	make stop-db
 
-coverage: venv_check install_dev
-	coverage run --source="." manage.py test
+coverage: venv_check install_dev start-db
+	ACG_CONFIG=cdappconfig.json coverage run --source="." manage.py test
```

no need since exporting `ACG_CONFIG` at the top.
```suggestion
	coverage run --source="." manage.py test
```